L/Libepoxy/build_tarballs.jl

```diff
@@ -4,25 +4,81 @@ using BinaryBuilder
 
 name = "Libepoxy"
 version = v"1.5.10"
+# We bumped the version number to build for riscv64
+ygg_version = v"1.5.11"
 
 # Collection of sources required to build Libepoxy
 sources = [
-    ArchiveSource("https://github.com/anholt/libepoxy/archive/refs/tags/$(version).tar.gz",
-                  "a7ced37f4102b745ac86d6a70a9da399cc139ff168ba6b8002b4d8d43c900c15")
+    GitSource("https://github.com/anholt/libepoxy", "c84bc9459357a40e46e2fec0408d04fbdde2c973"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libepoxy-*/
+cd $WORKSPACE/srcdir/libepoxy
+
+# meson unconditionally (?) adds the command line argument
+# `-Werror=unused-command-line-argument`. That would be fine, except
+# that we (the `cc` script) add `-Wl,-sdk_version,11.0` on Darwin,
+# which is actually unused when meson checks for supported compiler
+# options. This means that each such check fails, and meson won't use
+# any options. That's mostly fine, except for the option
+# `-Wno-int-conversion` which is required by the source code.
+#
+# The code below modifies the `cc` script to filter out the option
+# `-Werror=unused-command-line-argument`. This should arguably either
+# happen by default, or `cc` should only add the
+# `-Wl,-sdk_version,11.0` when the linker is actually called.
+#
+# We modify `cc` on all architectures to prevent possible similar
+# errors there.
```

Comment:
I'm moderately hopeful (but haven't tested) that adding `-Wno-unused-command-line-argument` to meson C/C++ flags would work around this issue in a less painful way.  If only Meson didn't expect to control _everything_ under the sun and left escape hatches for cases like this, it'd be a much nicer build system.

Reply:
But also, if this was meson-specific, we'd have this issue with all other meson-based packages, no?  What am I missing?

Reply:
```env CFLAGS='-Wno-unused-command-line-argument' meson .. -Dtests=false --buildtype=release --cross-file="${MESON_TARGET_TOOLCHAIN}"

Reply:
I briefly considered patching `cc` instead of modifying it by line number but that's even more tedious. First, the source of `cc` isn't readily available, one has to run a container to get the scripts. And second, this script is very architecture dependent so I'd have to create a different patch for each architecture. That patch is likely to break whenever the script is updated, and updating the patch is quite tedious (since the `cc` source code isn't available online).

Reply:
I'm modifying the meson target toolchain now. That seems to be the only way to make meson listen.

Reply:
> is not working

That's because meson completely ignores all environment variables. I've been telling you meson is stubborn.

Reply:
Do you have a different idea for addressing this?